src/index.md

```diff
@@ -242,7 +242,7 @@ options: `article`, `product`
 default: `website`
 
 ### url
-The content for `og:url` and `twitter:url`.
+The content for `<link rel=canonical href="foo.com">`, `og:url` and `twitter:url`.
```

Comment:
You forgot to address `rel=canonical` --> `rel="canonical"`

Reply:
👋 @wommy Just a heads up, this is sort of a "missing piece" in the docs while waiting for this PR to be fixed up. 

I need this documentation to appear on the site so I'm going to close this and update it myself if not addressed by tomorrow. Thanks for your help thus far.

Reply:
yo finished it!

i dont check everyday 🙇

Reply:
no worries! thanks for the fix